tests/pw/.env

```
@@ -0,0 +1,52 @@
+TAG_ID=31
+SERVER_URL=http://localhost:9999/wp-json
+CUSTOMER_ID=3
+VENDOR_ID=4
+CUSTOMER2_ID=5
+VENDOR2_ID=6
+PRODUCT_EDIT_NONCE=f1351ea962
+PRODUCT_ID=36
+PRODUCT_ID_V2=37
+CATEGORY_ID=32
+ATTRIBUTE_ID=2
+TAG_ID=41
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The TAG_ID key is duplicated

(DuplicatedKey)